### PR TITLE
add intro

### PR DESCRIPTION
**Pull request by juliangruber**
this is a WIP pull request, adding the intro skeleton for https://github.com/datproject/dat-desktop/issues/342 so @Kriesse can continue working on it.

### CODE REVIEW

**Comment by Kriesse** [comment]
@juliangruber Can you look into the failing tests? I think this is ready to merge, we can iterate on copy in a separate issue!

**Comment by juliangruber** [comment]
awesome, yes will do!

File: elements/intro.js [hunk 1]
```diff
@@ -85,101 +85,56 @@ function IntroScreen () {
...
+                  Check out <a href="https://datproject.org/" class="color-green-disabled hover-color-green">datproject.org</a> to explore open datasets.
+                </p>
+              `
+          }[screen]}
+        </div>
+        <div class="${footer}">
+          ${screen === 0
+            ? button.green('Get Started', { onclick: next })
+            : html`
+                <div>
```
Comment (juliangruber) [inline elements/intro.js]: @Kriesse this `<div>` messed up rendering a bit, can you check?

**Comment by juliangruber** [comment]
next build should be fixed, looks like something was messed up on travis

**Comment by Kriesse** [comment]
@juliangruber footer markup is cleaned up ✅

**Comment by juliangruber** [comment]
@Kriesse thanks! there's a z-index issue now though, the svg element lies over the buttons

**Comment by juliangruber** [comment]
i could add the `z-index` myself but maybe you'd rather choose to reposition the svg element instead?

**Comment by juliangruber** [comment]
the integration tests work again! :) shall we merge?

**Comment by Kriesse** [comment]
@juliangruber yes!

**Pull request merged**